stdlib/REPL/src/TerminalMenus/TerminalMenus.jl

```diff
@@ -23,4 +23,9 @@ export
     MultiSelectMenu,
     request
 
+# TODO: remove Julia 2.0
+# While not exported, AbstractMenu documented these as an extension interface
+const printMenu = printmenu
+const writeLine = writeline
```

Comment:
```suggestion
@deprecate printMenu printmenu
@deprecate writeLine writeline
```
It seems okay to me to introduce an actual deprecation for these since they aren't exported and TerminalMenus usage is fairly limit AFAIK.